client/python/test/test_serializing.py

```diff
@@ -1,12 +1,24 @@
 import os
 import tempfile
 
+import pytest
 from gradio import components
 
-from gradio_client.serializing import COMPONENT_MAPPING, FileSerializable
+from gradio_client.serializing import COMPONENT_MAPPING, FileSerializable, Serializable
 from gradio_client.utils import encode_url_or_file_to_base64
 
 
+@pytest.mark.parametrize("serializer_class", Serializable.__subclasses__())
+def test_duplicate(serializer_class):
```

Comment:
To make sure I understand, this basically tests to make sure all of the serializer classes have their info defined in the `types.json` file, correct?

Reply:
Yes!